source/fundamentals/collations.txt

```diff
@@ -4,5 +4,302 @@ Collations
+specify a collation in a subset of operations.
+
+Collection or View
+~~~~~~~~~~~~~~~~~~
+
+You can specify a collation when you create a new collection or view. This defines the default
+collation for any operations called on that collection or view. Set a collation through a 
+``CreateCollectionOptions`` or ``CreateViewOptions`` object. Then, call the
+``CreateCollection()`` or ``CreateView()`` method with your options object as an argument.
+
+Index
+~~~~~
+
+You can specify a collation when you create a new index on a collection. The index stores
+an ordered representation of the documents in the collection so your operation does not need
+to perform the ordering in-memory. 
+
+To use the index in an operation, your operation must specify the same collation as the one
+specified in the index. Additionally, ensure that the operation is covered by the index that
+contains the collation. Set a collation through an ``IndexOptions`` object and call the
+``CreateOne()`` method with your options object as an argument.
+
+Operation
+~~~~~~~~~
+
+Specifying a collation on an operation overrides any default collation previously
+defined for a collection. If you specify a new collation on an operation that differs
+from the index collation, you cannot use that index. As a result, the operation may not
+perform as well as one that is covered by an index. 
+
+For more information on the disadvantages of sorting operations not covered by an index,
+see the server manual page on :manual:`Using Indexes to Sort Query Results </tutorial/sort-results-with-indexes/>`.
+See the :manual:`MongoDB manual </reference/collation/#collation-document>` for a list of
+operations that support collation.
+
+Collation Examples
+------------------
+
+.. _golang-collection-collation:
+
+Set a Default Collation on a Collection
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The following example creates a new collection called ``books`` and specifies a default
+collation with the ``"fr"`` locale. The ``Strength`` collation field has a value of ``1``
+to ignore differences in letter accents.
+
+.. code-block:: go
+
+   myCollation := &options.Collation{Locale: "fr", Strength: 1}
+   opts := options.CreateCollection().SetCollation(myCollation)
+   err := db.CreateCollection(context.TODO(), "books", opts)
+
+   if err != nil {
+      log.Fatal(err)
+   }
+
+Use the Default Collection Collation
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+If you call an operation that supports collation on the ``books`` collection, the operation
+will now use the default collation. Assume you want to insert the following documents
+into ``books``:
+
+.. code-block:: json
+
+   {"name" : "Emma", "author" : "Austen", "length" : "474"}
+   {"name" : "Les Misérables", "author" : "Hugo", "length": "1462"}
+   {"name" : "Infinite Jest", "author" : "Wallace", "length" : "1104"}
+   {"name" : "Cryptonomicon", "author" : "Stephenson", "length" : "918"}
+   {"name" : "Ça", "author" : "King", "length" : "1138"}
```

Comment:
Issue: As the author field is not used in examples, I believe it should be deleted (adds cognitive load without helping to illustrate concepts of the page).

Reply:
I took out the author field